### PR TITLE
Cdash followups

### PR DESCRIPTION
**Pull request by drwells**
I added the copyright attribution to the new cmake files and also changed keywords to be all caps. I know its deprecated but I am stuck in my ways on this point - if we have them in all caps we are at least internally consistent.

@vibraphone I very strongly doubt that this is an issue but does this copyright attribution cause any problems? We use the same copyright policy as Linux (documented [here](https://github.com/IBAMR/IBAMR/blob/master/CONTRIBUTING.md))

### IBAMR Pull Request Checklist
- [x] Does the test suite pass?
- [x] Was clang-format run?
- [x] Were relevant issues cited? Please remember to add `Fixes #12345` to close
      the issue automatically if we are fixing the problem.
- [x] Is this a change others will want to know about? If so, then has a
      changelog entry been added?
- [x] If new data structures have been added to a class then have they been set
      up appropriately for restarts? If so, ensure that the restart version
      number is incremented.
- [x] Does this change include a bug fix or new functionality? If so, a new test
      or tests should be added. New tests should run quickly (less than a minute
      in release mode). If possible, an older test should gain a new option so
      that we do not need to compile more test executables.
- [x] Did you (if your account has permission to do so) set relevant labels on
      GitHub for the pull request?

### CODE REVIEW

**Comment by vibraphone** [comment]
No problems with the copyrights/style.

File: .github/cmake/github_ci.cmake [hunk 1]
```diff
@@ -1,47 +1,60 @@
...
-  set(ctest_track "merge-requests")
-elseif ("$ENV{GITHUB_EVENT_NAME}" STREQUAL "push")
+  SET(ctest_track "merge-requests")
+ELSEIF("$ENV{GITHUB_EVENT_NAME}" STREQUAL "push")
   # Put all branch tests in a track named the same as the branch.
   # Currently, only master is tested, but eventually branch names
   # such as "release-0.9" may exist and should have their own tracks.
-  string(REGEX_REPLACE "refs\\/heads\\/" "" ctest_track "$ENV{GITHUB_REF}")
-endif ()
+  STRING(REGEX_REPLACE "refs\\/heads\\/" "" ctest_track "$ENV{GITHUB_REF}")
```
Comment (vibraphone) [inline .github/cmake/github_ci.cmake:59]:
```suggestion
  STRING(REGEX REPLACE "refs\\/heads\\/" "" ctest_track "$ENV{GITHUB_REF}")
```

Comment (vibraphone) [reply .github/cmake/github_ci.cmake:59]: This change will fix the "on-push" action failures when PRs are merged to a branch.

**Review by vibraphone — CHANGES_REQUESTED** [review]
The suggested change fixes an error on my part. The additional test when a branch is merged
+ checks for failures if the target branch (usu. `origin/master`) has changed since the PR base commit
+ submits tests to the dashboard under the branch name (rather than "experimental"); the branch tests are held over from day to day even if no new merges occur (so the dashboard won't be empty just because there hasn't been activity in the last day).

File: .github/cmake/github_ci.cmake (hunk 1 above)
Comment (drwells) [reply .github/cmake/github_ci.cmake]: Nice catch. I'm surprised that didn't error out in a more visible way.

**Comment by drwells** [comment]
This stuff has all been fixed in other PRs by this point, so we can close this.

**Pull request closed without merging**